cli/src/build/onboarding/android/ui/app.tsx

```
@@ -0,0 +1,1798 @@
+          <Newline />
+          <Text dimColor>The digits after <Text color="cyan">/developers/</Text> are what we need. Copy them, or copy the whole URL — we&apos;ll parse it.</Text>
+          <Newline />
+          <Select
+            options={[
+              { label: '🌐  Open Play Console in my browser', value: 'open' },
+              { label: '🎬  Watch a quick video tutorial', value: 'tutorial' },
+              { label: '📝  I have my developer ID — let me paste it', value: 'manual' },
+            ]}
+            onChange={async (value) => {
+              if (value === 'open') {
+                try { await open(PLAY_DEVELOPERS_URL) }
```

ESLint

[error] This line has 2 statements. Maximum allowed is 1.

(style/max-statements-per-line)